wrong ldap_sync_destroy() prototype in ldap_sync(3) manpage

Author: Jan Vcelak <jvcelak@redhat.com>
Resolves: #717722

---
 doc/man/man3/ldap_sync.3 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/doc/man/man3/ldap_sync.3 b/doc/man/man3/ldap_sync.3
index ef15085..e2c0880 100644
--- a/doc/man/man3/ldap_sync.3
+++ b/doc/man/man3/ldap_sync.3
@@ -20,7 +20,7 @@ OpenLDAP LDAP (libldap, \-lldap)
 .LP
 .BI "ldap_sync_t * ldap_sync_initialize(ldap_sync_t *" ls ");"
 .LP
-.BI "int ldap_sync_destroy(ldap_sync_t *" ls ", int " freeit ");"
+.BI "void ldap_sync_destroy(ldap_sync_t *" ls ", int " freeit ");"
 .LP
 .BI "typedef int (*" ldap_sync_search_entry_f ")(ldap_sync_t *" ls ","
 .RS
-- 
1.7.6

